bin/parse-sld.py

```diff
@@ -0,0 +1,101 @@
+# parse-sld.py
+#
+# This script takes in an SLD file, and converts its output to MapServer
+# CLASS expressions. It understands the topographic SLD files from the
+# OSMM-Topography-Layer-stylesheets repository.
+
+import sys
+from lxml import objectify
+
+line_class = '''
+CLASS
+  EXPRESSION "%(expression)s"
+  STYLE
+    COLOR "%(color)s"
+    MINWIDTH 0.2
+    WIDTH %(width)s
+    %(pattern)s
+  END
+END
+'''
+
+poly_class = '''
+CLASS
+  EXPRESSION "%(expression)s"
+  STYLE
+    COLOR "%(color)s"
+    WIDTH 0
+  END
+END
+'''
+
+symbol_class = '''
+CLASS
+  EXPRESSION "%(expression)s"
+  STYLE
+    COLOR "%(color)s"
+  END
+  STYLE
+    SYMBOL %(symbol)s
+    SIZE %(size)s
+    WIDTH 0
+  END
+END
+'''
+
+hatch_class = '''
+CLASS
+  EXPRESSION "%(expression)s"
+  STYLE
+    SYMBOL "hatchsymbol"
+    COLOR "%(color)s"
+    SIZE %(size)s
+    WIDTH %(width)s
+    ANGLE %(angle)s
+  END
+END
+'''
+
+tree = objectify.parse(open(sys.argv[1]))
+styles = tree.getroot().NamedLayer.UserStyle
+for f in styles.iterchildren(tag='{http://www.opengis.net/sld}FeatureTypeStyle'):
+    rule = f.Rule
+    args = {
+        'expression': rule['{http://www.opengis.net/ogc}Filter'].PropertyIsEqualTo.Literal.text,
+    }
+
+    if getattr(rule, 'PolygonSymbolizer', None) is not None:
+        fill = rule.PolygonSymbolizer.Fill
+        if getattr(fill, 'CssParameter', None):
+            args['color'] = fill.CssParameter.text
+            try:
+                graphic = rule.PolygonSymbolizer[1].Fill.GraphicFill.Graphic
+                args['symbol'] = graphic.ExternalGraphic.OnlineResource.get('{http://www.w3.org/1999/xlink}href')
+                args['size'] = graphic.Size
+                print symbol_class % args
```

Comment:
If this was `print(symbol_class % args)` (and `python3 -m venv "$VENV_DIR"` instead of `virtualenv "$VENV_DIR"` in `load-mm`) then this would work in Python 3.